staging/src/k8s.io/kubectl/pkg/cmd/apply/apply_test.go

```diff
@@ -54,7 +54,7 @@ import (
 )
 
 var (
-	fakeSchema                 = sptest.Fake{Path: filepath.Join("..", "..", "..", "..", "api", "openapi-spec", "swagger.json")}
```

Comment:
was fakeSchema pointing at the real static schema before?

Reply:
Yes. There are several instances of this in the tests.